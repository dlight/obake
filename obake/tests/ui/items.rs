#[obake::versioned]
fn foo() {}

fn main() {}
