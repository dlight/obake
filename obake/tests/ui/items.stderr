error: expected `struct` or `enum`
 --> $DIR/items.rs:2:1
  |
2 | fn foo() {}
  | ^^
